wen-restart/protos/wen_restart.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+package solana.wen_restart_proto;
+
+message WenRestartProgress {
+    enum State {
+        INIT = 0;
+        LAST_VOTED_FORK_SLOTS = 1;
+        HEAVIEST_FORK = 2;
+        GENERATING_SNAPSHOT = 3;
+        FINISHED_SNAPSHOT = 4;
+        WAITING_FOR_SUPERMAJORITY = 5;
+        DONE = 6;
+    }
+
+    message InitRecord {
```

Comment:
It's not totally clear to me what an InitRecord is. Might be more clear if you spell out Init? I'm not sure if Init is a noun or an adjective (or a verb, I suppose).

Reply:
Renamed to MyLastVotedForkSlots.